### PR TITLE
Feature - Upgrade react-scripts to v3

### LINKED ISSUE #131: Upgrade scripts to cra 3.0.0

### PR DESCRIPTION
**Pull request by apanizo**
**Description**
This PR updates the react-scripts to version 3, directly in the library for bierzo-wallet project and also the separate fork for enabling chrome extensions development (I have updated my own project [see here](https://github.com/apanizo/create-react-app/pull/2)).

Now we have proper linting reports when using hooks:
```
yarn run v1.15.2
$ eslint -c .eslintrc.js 'src/**/*.ts{,x}'

/Users/apanizo/git/iov/ponferrada/packages/medulas-react-components/src/components/forms/SelectFieldForm/index.tsx
  86:6  warning  React Hook React.useEffect has missing dependencies: 'initial', 'onChange', and 'value'. Either include them or remove the dependency array  react-hooks/exhaustive-deps

✖ 1 problem (0 errors, 1 warning)
  0 errors and 1 warning potentially fixable with the `--fix` option.
```

This PR closes #131 

**TODO**
- [x] Fix eslint new usage. See breaking changes part of [this](https://github.com/facebook/create-react-app/releases)
- [x] Allow react-scripts 3.0 on bierzo-wallet
- [x] Check HMR is working on the chrome extension
- [x] Update storybook libraries for making everything to work

### CODE REVIEW

File: packages/bierzo-wallet/src/routes/welcome/components/ExtensionInteraction.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+/*global chrome*/
```
Comment (albertandrejev) [inline packages/bierzo-wallet/src/routes/welcome/components/ExtensionInteraction.tsx]: Can you tell me please, what is this for?

File: packages/sanes-chrome-extension/src/utils/balances/index.ts [hunk 2]
```diff
@@ -6,7 +6,7 @@ export type Figures = Omit<Amount, 'tokenTicker'>;
 export function parseFigures(amount: string): Figures {
   // trim off all leading zeros when parsing
   const trimmed = amount.replace(/^0+/, '');
-  const matched = trimmed.match(/^([0-9]+)?([\.\,])?([0-9]+)?$/);
+  const matched = trimmed.match(/^([0-9]+)?([.,])?([0-9]+)?$/);
```
Comment (webmaster128) [inline packages/sanes-chrome-extension/src/utils/balances/index.ts]: Most likely, this diff creates a new bug. `.` means any character while `\.` means dot (which we need here). The `,` does not need to be escaped but the `.` must.

File: packages/bierzo-wallet/src/routes/welcome/components/ExtensionInteraction.tsx (hunk 1 above)
Comment (webmaster128) [reply packages/bierzo-wallet/src/routes/welcome/components/ExtensionInteraction.tsx]: It tells eslint that a global called `chrome` exists (https://eslint.org/docs/user-guide/configuring#specifying-globals).

File: packages/sanes-chrome-extension/src/utils/balances/index.ts (hunk 2 above)
Comment (webmaster128) [reply packages/sanes-chrome-extension/src/utils/balances/index.ts]: Sorry, I was wrong in this case. Just double checked this: inside of a character classe (between `[` and `]`), the `.` litterally means dot. So this change is perfectly fine.

**Pull request merged**